src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -3020,1617 +3018,6 @@ interface unmanaged {}
                 Keyword("unmanaged"));
         }
 
-        [Theory, CombinatorialData]
-        public async Task TestRegex1(TestHost testHost)
```

Comment:
i extracted out embedded classification tests to their own dedicated files.  This file is just the tests that were moved out.